doc/tutorials/01_running_blink.md

```diff
@@ -58,10 +58,11 @@ compiled and loaded much like the kernel is. See the [getting started README](..
 
     ```bash
     cd userland/examples/blink
-    make program  # Load code via bootloader
+    make TOCK_BOARD=imix program  # Load code via bootloader
```

Comment:
This won't work for Hail. Do we need to clarify in these tutorials if that we're talking specifically about `imix`? Is that even what we want at this point, when the only hardware most people can get is a Hail?

@bradjc thoughts? I think you wrote these tutorials mostly. I wonder if they're not better off on the website where we can add some sort of choose-your-own-adventure based on which board the reader is using.

Reply:
To be fair, the README does note "If you have another board than Imix relace imix with your board." I assume that running `make TOCK_BOARD=hail program` would work, even if it is unnecessary to include the extra argument for the hail board? 

Given that even before any of the changes I made, the first step of the tutorial was this:
![image](https://user-images.githubusercontent.com/32688905/31478382-d05c4f04-aec4-11e7-970b-a4bc368d6a58.png)
It seems as though asking users to determine and specify their Tock Board is already part of the tutorial.  Given that most people can only obtain a Hail, I agree it might make more sense to leave the format as I made it but change every reference to `imix `in the tutorial to say `hail`. This would result in the tutorial still being oriented towards hail users, while also still presenting instructions which imix users could easily follow and get to work. This seems preferable to a tutorial which just uses `make program` to save Hail users from having to type an extra argument but that Imix users cannot follow.

Reply:
Lol... OK, you're right :)

Reply:
The long term plan (or as soon as possible ideally) is to remove the `TOCK_BOARD` variable so this won't be an issue. I think if we going to put any effort in anywhere it should be towards getting the bootloader written in Tock and kernel-level attributes working, which will enable removing `TOCK_BOARD`.